log/aix_utmp.ksy

```diff
@@ -0,0 +1,75 @@
+meta:
+  id: aix_utmp
+  title: utmp log file, IBM AIX version
+  license: CC0-1.0
+  endian: be
+doc: |
+  This spec can be used to parse utmp, wtmp and other similar as created by
+  IBM AIX. Spec is based on IBM Knowledge Base article:
+  https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/com.ibm.aix.files/utmp.h.htm
+seq:
+  - id: records
+    type: record
+    repeat: eos
+types:
+  record:
+    seq:
+    - id: ut_user
+      doc: User login name
+      size: 256
+      type: str
+      encoding: utf-8
```

Comment:
move into `meta`

Reply:
I actually seriously doubt that it's always utf-8 — I'm not even sure if AIX supports utf-8, or it is a default encoding in AIX 7.2. May be we should just let it stay without encoding?